src/js/webrequest.js

```diff
@@ -776,6 +782,14 @@ function dispatcher(request, sender, sendResponse) {
     badger.storage.getBadgerStorageObject("action_map").deleteItem(request.origin);
 
     sendResponse();
+  } else if (request.type == "checkDNT") {
```

Comment:
This either has to be `} else if (request.checkDNT) {`, or you have to update the message you send from the content script.

Reply:
Style nit: Could you add a blank line above the "else if"?

Reply:
Good catch. Fixed.